src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js

```diff
@@ -76,11 +83,81 @@ export const ParallelCoordinates = ({ metricsData, selectedRuns }) => {
   };
 
   const handleMouseOverMetric = (e, key) => {
+    const runsCount = graph.find((each) => each[0] === key)[1].length;
     setHoveredAxisG(key);
+
+    const rect = e.target.getBoundingClientRect();
+
+    let x, direction;
+
+    if (window.innerWidth - rect.x > tooltipMaxWidth) {
+      x = e.clientX - sideBarWidth;
+      direction = 'right';
+    } else {
+      x = e.clientX - sideBarWidth - sideBarWidth / 2;
+      direction = 'left';
+    }
+    const y = rect.y - 140;
```

Comment:
```suggestion
    const rect = e.target.getBoundingClientRect();
    const y = rect.y - 140;
    let x, direction;

    if (window.innerWidth - rect.x > tooltipMaxWidth) {
      x = e.clientX - sideBarWidth;
      direction = 'right';
    } else {
      x = e.clientX - sideBarWidth - sideBarWidth / 2;
      direction = 'left';
    }
```

Also where does the `140` value come from here: `const y = rect.y - 140;`?